mev-build-rs/src/reth_builder/payload_builder.rs

```diff
@@ -274,6 +274,8 @@ impl<'a> ExecutionContext<'a> {
             success: result.is_success(),
             cumulative_gas_used: self.cumulative_gas_used,
             logs: result.logs().into_iter().map(into_reth_log).collect(),
+            #[cfg(feature = "optimism")]
+            deposit_nonce: None,
```

Comment:
very interested in supporting the optimism use case, although I don't know enough about their eco to say what is needed here and what the demand for PBS is

can you point me to resources expounding on these points?